server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -1126,18 +1174,30 @@
   "MarketValue_Farm_Crops": "Market Value of Farms Growing Crops",
   "MarketValue_Farm_LivestockAndPoultry": "Market Value of Livestock and Poultry on a Farm",
   "MarketValue_Farm_MachineryAndEquipment": "Market Value of Farm Machinery and Equipment",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_Historical": "Max Temperature With 1% Chance Once In Decade: CMIP6 Ensemble, Historical",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": "Max Temperature With 1% Chance Once In Decade: CMIP6 Ensemble, SSP245",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585": "Max Temperature With 1% Chance Once In Decade: CMIP6 Ensemble, SSP585",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_Historical": "Max Temperature With 1% Chance Once In Year: CMIP6 Ensemble, Historical",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_SSP245": "Max Temperature With 1% Chance Once In Year: CMIP6 Ensemble, SSP245",
-  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_SSP585": "Max Temperature With 1% Chance Once In Year: CMIP6 Ensemble, SSP585",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_Historical": "Max Temperature With 5% Chance Once In Decade: CMIP6 Ensemble, Historical",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": "Max Temperature With 5% Chance Once In Decade: CMIP6 Ensemble, SSP245",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585": "Max Temperature With 5% Chance Once In Decade: CMIP6 Ensemble, SSP585",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_Historical": "Max Temperature With 5% Chance Once In Year: CMIP6 Ensemble, Historical",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_SSP245": "Max Temperature With 5% Chance Once In Year: CMIP6 Ensemble, SSP245",
-  "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_SSP585": "Max Temperature With 5% Chance Once In Year: CMIP6 Ensemble, SSP585",
+  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_Historical": "Max Temperature (\u00b0C) Once In Decade With 1% Chance: Historical",
+  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": "Max Temperature (\u00b0C) Once In Decade With 1% Chance: SSP245",
+  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585": "Max Temperature (\u00b0C) Once In Decade With 1% Chance: SP585",
+  "MaxTemp_Daily_Hist_1PctProb_Greater_Atleast1DayAYear_CMIP6_Ensemble_Historical": "Max Temperature (\u00b0C) Once In Year With 1% Chance: Historical",
```

Comment:
nit: I assume this shows up right on UI?

Reply:
Thanks, yes they do. See: https://screenshot.googleplex.com/7KWevjghv5EhArU